requirements/FAULT_DETECTION_AURORA-0006/broker-error/work/taste.pro

```diff
@@ -0,0 +1,7 @@
+INCLUDEPATH += work/dataview/C
```

Comment:
I am fully aware that the validation method is M, but, for the sake of not having proper integration tests elsewhere, could we add a little sh script which pushes something over the socket to Node2 so that the error is printed, as we discussed?

Reply:
Added test.sh